[0.010s] Invoking command in '/home/praveen/udmey/build/my_robot_interfaces': /usr/bin/cmake /home/praveen/udmey/src/my_robot_interfaces -DCMAKE_INSTALL_PREFIX=/home/praveen/udmey/install/my_robot_interfaces
[0.022s] -- Found ament_cmake: 1.3.9 (/opt/ros/humble/share/ament_cmake/cmake)
[0.125s] -- Found rosidl_default_generators: 1.2.0 (/opt/ros/humble/share/rosidl_default_generators/cmake)
[0.126s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[0.128s] -- Found rosidl_adapter: 3.1.5 (/opt/ros/humble/share/rosidl_adapter/cmake)
[0.151s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[0.311s] [31mCMake Error at /opt/ros/humble/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:59 (message):
[0.311s]   execute_process(/usr/bin/python3 -m rosidl_adapter --package-name
[0.311s]   my_robot_interfaces --arguments-file
[0.311s]   /home/praveen/udmey/build/my_robot_interfaces/rosidl_adapter__arguments__my_robot_interfaces.json
[0.311s]   --output-dir
[0.311s]   /home/praveen/udmey/build/my_robot_interfaces/rosidl_adapter/my_robot_interfaces
[0.311s]   --output-file
[0.311s]   /home/praveen/udmey/build/my_robot_interfaces/rosidl_adapter/my_robot_interfaces.idls)
[0.311s]   returned error code 1:
[0.311s] 
[0.311s]   Traceback (most recent call last):
[0.311s] 
[0.312s]     File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
[0.312s]       return _run_code(code, main_globals, None,
[0.312s]     File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
[0.312s]       exec(code, run_globals)
[0.312s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__main__.py", line 19, in <module>
[0.312s]       sys.exit(main())
[0.312s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/main.py", line 53, in main
[0.312s]       abs_idl_file = convert_to_idl(
[0.312s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__init__.py", line 24, in convert_to_idl
[0.312s]       return convert_srv_to_idl(
[0.312s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/srv/__init__.py", line 28, in convert_srv_to_idl
[0.312s]       srv = parse_service_string(package_name, input_file.stem, content)
[0.312s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 859, in parse_service_string
[0.312s]       request_message = parse_message_string(
[0.312s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 544, in parse_message_string
[0.312s]       msg = MessageSpecification(pkg_name, msg_name, fields, constants)
[0.312s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 388, in __init__
[0.312s]       self.base_type = BaseType(
[0.313s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 205, in __init__
[0.313s]       raise InvalidResourceName(
[0.313s] 
[0.313s]   rosidl_adapter.parser.InvalidResourceName: 'area_Request' is an invalid
[0.313s]   message name.  It should have the pattern '^[A-Z][A-Za-z0-9]*$'
[0.313s] 
[0.313s] Call Stack (most recent call first):
[0.313s]   /opt/ros/humble/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
[0.313s]   CMakeLists.txt:17 (rosidl_generate_interfaces)
[0.313s] 
[0.313s] [0m
[0.313s] -- Configuring incomplete, errors occurred!
[0.313s] See also "/home/praveen/udmey/build/my_robot_interfaces/CMakeFiles/CMakeOutput.log".
[0.317s] Invoked command in '/home/praveen/udmey/build/my_robot_interfaces' returned '1': /usr/bin/cmake /home/praveen/udmey/src/my_robot_interfaces -DCMAKE_INSTALL_PREFIX=/home/praveen/udmey/install/my_robot_interfaces
